hackathon/templatetags/my_tags.py

```diff
@@ -2,6 +2,8 @@
 # range snippet from: https://www.djangosnippets.org/snippets/1357/
 # adjusted to current project needs based on https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/
 from django.template import Library
+import datetime
```

Comment:
As per PEP8 standard libraries should be imported first, then 3rd party libraries (e.g. django libraries) and they should be separated by a blank line.

Reply:
done